README.md

```diff
@@ -1,6 +1,12 @@
-# needle
+# Needle
 
-Needle is a dependency injection system for Swift iOS apps. It utilizes code generation to ensure compile-time safey. If it compiles, it works.
+Needle is a dependency injection system for Swift iOS apps. Unlike other iOS DI frameworks, such as [Cleanse](https://github.com/square/Cleanse), [Swinject](https://github.com/Swinject/Swinject), Needle encourages **hierarchical DI structure and utilizes code generation to ensure compile-time safey**. This allows us to develop our apps and making changes with confidence. **If it compiles, it works.** In this aspect, Needle is more similar to [Android Dagger](https://google.github.io/dagger/).
```

Comment:
safety*

Reply:
^ Does this make me a Needle contributor now?

Reply:
Haha yes it does. Welcome!

Reply:
... "and make changes with confidence.".  

nit: maybe "in this regard" instead of "in this aspect" ?